README.md

```diff
@@ -8,8 +8,11 @@ A bot that ensures exactly one newcomer issue is assigned per person so that it
 
 1. Whenever a user comments on any issue, it will be detected using “issue_comment” event.
 2. The label of the issue is returned in the payload response of the said event.
-3. We will create a configurable json file containing an array of labels( For e.g. `first-timers-only`) used for tagging new comer issues.
+3. We will create a configurable json file containing an array of labels( For e.g. `first-timers-only`, `new-comers-issue`) used for tagging new comer issues.
 4. If the issue label matches any of of the array entries , then further action will be taken.
-5. Using fuzzy logic in natural language processing on some standard phrases we will detect if the commenter is trying to claim the issue. We might use npm module nlp-toolkit for the same.
-6. If so, then using the user details from payload returned on the issue_comment event, we’ll search all the issues of the [repository](https://developer.github.com/v3/issues/#list-issues-for-a-repository), for the detected label to check if the commenter is assigned one.
+5. The comment is tested against a set of strings like "I want to work on this issue", "Dibs on this one!", "I wanna contribute", etc.
```

Comment:
It would be nice if you could point to the source code for this. Plus also add a link to create an issue for adding new strings into the array.

Reply:
Done!